TribalWars/src/Infrastructure/Configuration/BuildingRequiredConfig.cs

```diff
@@ -0,0 +1,27 @@
+﻿using ApplicationCore.Entities;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.EntityFrameworkCore.Metadata.Builders;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Infrastructure.Configuration
+{
+    class BuildingRequiredConfig : IEntityTypeConfiguration<RequiredBuilding>
```

Comment:
Entity has name "RequiredBuilding" and configuration class has name "BuildingRequired", this is not consistent.